stdlib/Random/src/Random.jl

```diff
@@ -359,4 +368,16 @@ true
 """
 srand(rng::AbstractRNG, ::Nothing) = srand(rng)
 
+
+## deprecations
```

Comment:
The other stdlibs have an `deprecated.jl` file that they `include`.

Reply:
OK thanks